package.json

```diff
@@ -24,11 +24,17 @@
   },
   "homepage": "https://github.com/slashauth/slashauth-node-client#readme",
   "devDependencies": {
+    "@types/express": "^4.17.14",
     "@types/node": "^17.0.40",
+    "@types/react": "^18.0.20",
+    "@types/react-dom": "^18.0.6",
+    "encoding": "^0.1.13",
     "typescript": "^4.7.3"
   },
   "dependencies": {
     "crypto": "^1.0.1",
+    "express": "^4.18.1",
```

Comment:
This seems a bit dicy.

This requires the person installs express (which they may not have). It can be better to use `peer-dependencies` here. But then that gets dicy too because they might use like Koa and then the peer dependencies will fail.

Any issues with conflicting express versions?